first few lines for a vgg script

tristaaan opened the pull request.

diegocasmo reviewed (changes requested):
- [ ] Define ``history,checkpointer,early_stopping`` callbacks
- [ ] Draw RMSE plot

tristaaan committed:
vgg initial work ...
this is the remains of a branch that got horribly mangled after a bad rebase, the important commits from their respective authors are listed as follows:

- (tw) first few lines for a vgg script
- (as) initial vgg16 prediction and bulk of preprocessing
- (as) get train data by filename
- (tw) vgg-16 with gpu, cuda test
- (tw) added cuda test, and vgg-16 which uses gpu

tristaaan committed:
history and loss functions
